src/docs/user.docs.js

```diff
@@ -162,3 +162,24 @@
  *          200:
  *              description: Success
  */
+
+/**
+ * @swagger
+ * /users/disable:
+ *  patch:
+ *    tags:
+ *      - Authentication
+ *    summary: Disables a User
+ *    description: User Can Disable his account
+ *    responses:
```

Comment:
This swagger route needs `security: - bearerAuth: []`. otherwise it won't work.

Reply:
changes resolved